test/foundry/BlastPoints.t.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {OwnableTwoSteps} from "../../contracts/OwnableTwoSteps.sol";
+import {BlastPoints} from "../../contracts/BlastPoints.sol";
+import {TestHelpers} from "./utils/TestHelpers.sol";
+
+import {MockBlastPoints} from "../mock/MockBlastPoints.sol";
+
+contract BlastPointsOwnableTwoSteps is BlastPoints, OwnableTwoSteps {
```

Comment:
you actually don't need this, you can use BlastPoints

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/666c866d0326865e1a1e8178aca6a9452a714be3